README.md

```diff
@@ -0,0 +1,2 @@
+# food
+The food application
```

Comment:
I agree in General but in this case nobody is going to deploy it but me and its quite simple